### ISSUE TITLE
Grafana Some Panel No Data

### ISSUE THREAD

**Issue by huzk8** [juicedata/juicefs#2525]
**What happened**:
Some panel appear no Data.
<img width="1355" alt="image" src="https://user-images.githubusercontent.com/18548053/184586999-e513a9d8-1dbc-4f37-9c27-70f884c3ba15.png">
In prometheus , i can find the detail metrics data ,like below.
![image](https://user-images.githubusercontent.com/18548053/184587357-19e236f1-1c1d-4e88-96fd-052eb0f72483.png)

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**

**Environment**:
- JuiceFS version (use `juicefs --version`) or Hadoop Java SDK version:
- Cloud provider or hardware configuration running JuiceFS:
- OS (e.g `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Object storage (cloud provider and region, or self maintained):
- Metadata engine info (version, cloud provider managed or self maintained):
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage):
- Others:

**Comment by zhijian-pro** [comment]
Whether the time range is too small and no I/O operations are performed during this period？

**Comment by chnliyong** [comment]
The **Operations** panel's metrics is `juicefs_fuse_xxx` like, not `juicefs_object_xxx`. 
You can query `juicefs_fuse_written_size_bytes_sum` or `juicefs_fuse_read_size_bytes_sum` in your prometheus

**Comment by huzk8** [comment]
> The **Operations** panel's metrics is `juicefs_fuse_xxx` like, not `juicefs_object_xxx`. You can query `juicefs_fuse_written_size_bytes_sum` or `juicefs_fuse_read_size_bytes_sum` in your prometheus

the Operations panel edit page is below:
![image](https://user-images.githubusercontent.com/18548053/184593971-84fddde6-fabc-4922-84c1-7ef3dce90c66.png)


i query ` juicefs_fuse_ops_durations_histogram_seconds_count` in promethus, the data is 0 . is it normal ?
![image](https://user-images.githubusercontent.com/18548053/184593904-eacccc7f-4dd9-489e-87da-5d8b4866e2e0.png)

**Comment by huzk8** [comment]
> Whether the time range is too small and no I/O operations are performed during this period？

past five miniute , those no data panel still no data .
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/18548053/184594416-1a5272ed-a239-4031-a30f-786b7b58e151.png">

**Comment by chnliyong** [comment]
These panels display the fuse metrics, you don't mount it for use (Hadoop SDK mode) and some fuse metrics like fuse read and fuse write don't exist.

**Comment by huzk8** [comment]
> These panels display the fuse metrics, you don't mount it for use (Hadoop SDK mode) and some fuse metrics like fuse read and fuse write don't exist.

i just use juicefs hadoop sdk ， so the some panel  appear no data is normal ？

**Comment by chnliyong** [comment]
> > These panels display the fuse metrics, you don't mount it for use (Hadoop SDK mode) and some fuse metrics like fuse read and fuse write don't exist.
> 
> i just use juicefs hadoop sdk ， some panel appear no data is normal ？ if it is noraml , i will close this issue later.

Yes, you can mount it and do some read or write in the mounted directory, then these panels should be OK.

**Comment by davies** [comment]
> > These panels display the fuse metrics, you don't mount it for use (Hadoop SDK mode) and some fuse metrics like fuse read and fuse write don't exist.
> 
> i just use juicefs hadoop sdk ， some panel appear no data is normal ？ if it is noraml , i will close this issue later.

The prometheus can't pull metrics from Hadoop SDK, you need to use the P8S push gateway or graphite to push metrics from Hadoop SDK.
